action.yaml

```diff
@@ -29,17 +29,13 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: actions/checkout@v4
```

Comment:
Note that this action is not a workflow like the ones under ..github/workflows`. It provides an action that can be used from [other repositories' workflows](https://github.com/os-autoinst/qa-tools-backlog-assistant/blob/master/.github/workflows/backlog_checker.yml#L21).
It needs to do a checkout into a different directory.
I don't know right now how to solve the problem, but this won't work.

Reply:
I would guess `ref: ${{ github.event.pull_request.head.sha }}` is needed here. Unfortunately GitHub docs don't seem to explain this case?

Someone on stackoverflow suggests `github.event.workflow_run.head_sha` would work without giving an explanation. And maybe `fetch-depth: 2` is needed?

I'm thinking out loud here since that's what I had to look up myself

Reply:
seems to work now but not sure how affects https://github.com/os-autoinst/qa-tools-backlog-assistant/blob/master/.github/workflows/backlog_checker.yml#L21